Marlin/Conditionals_LCD.h

```diff
@@ -101,7 +110,8 @@
   #if ENABLED(ULTIMAKERCONTROLLER)              \
    || ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) \
    || ENABLED(G3D_PANEL)                        \
-   || ENABLED(RIGIDBOT_PANEL)
+   || ENABLED(RIGIDBOT_PANEL)                   \
+   || ENABLED(REPRAPWORLD_KEYPAD)
```

Comment:
NEWPANEL is/was already included with REPRAPWORLD_KEYPAD:
https://github.com/MarlinFirmware/Marlin/blob/bugfix-1.1.x/Marlin/Conditionals_LCD.h#L109

Are you thinking that ANET_KEYPAD_LCD should define NEWPANEL instead of REPRAPWORLD_KEYPAD?  I can test that out when my print finishes...